packages/core/test/inspect.test.ts

```diff
@@ -6,36 +6,45 @@ import {
   sendTo,
   waitFor,
   InspectionEvent,
-  isMachineSnapshot
+  isMachineSnapshot,
+  assign,
+  ContextFrom,
+  EventObject,
+  AnyTransitionDefinition,
+  raise
 } from '../src';
 
-function simplifyEvent(inspectionEvent: InspectionEvent) {
-  if (inspectionEvent.type === '@xstate.event') {
-    return {
-      type: inspectionEvent.type,
-      sourceId: inspectionEvent.sourceRef?.sessionId,
-      targetId: inspectionEvent.actorRef.sessionId,
-      event: inspectionEvent.event
-    };
-  }
-  if (inspectionEvent.type === '@xstate.actor') {
-    return {
-      type: inspectionEvent.type,
-      actorId: inspectionEvent.actorRef.sessionId
-    };
-  }
+function simplifyEvents(inspectionEvents: InspectionEvent[]) {
```

Comment:
I'd recommend adding a `filter` parameter here or something and using this utility across the whole test file. This way every test case will be structured in the same way - and that makes it easier to copy-paste things around or just make bulk changes to this file.